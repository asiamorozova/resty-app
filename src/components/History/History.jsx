import React from 'react';
import PropTypes from 'prop-types';

const History = ({ url, method }) => (
  <div>
    <p>{url}</p>
    <p>{method}</p>
  </div>
);
History.propTypes = {
  url: PropTypes.string.isRequired,
  method: PropTypes.string.isRequired
};

export default History;
