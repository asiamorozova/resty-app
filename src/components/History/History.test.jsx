import React from 'react';
import { shallow } from 'enzyme';
import History from './History';

describe('History component', () => {
  it('renders History', () => {
    const wrapper = shallow(<History
      url="https://testurl.com" 
      method='GET'/>);
    expect(wrapper).toMatchSnapshot();
  });
});
