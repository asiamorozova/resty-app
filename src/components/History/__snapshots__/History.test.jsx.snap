// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History component renders History 1`] = `
<div>
  <p>
    https://testurl.com
  </p>
  <p>
    GET
  </p>
</div>
`;
